lib/tmuxinator/config.rb

```diff
@@ -123,20 +126,43 @@ def directories
         end
       end
 
+      def valid_project_config?(project_config)
```

Comment:
Totally not relevant to this PR: we should prolly think about moving these validations to a separate file/module.  Getting a bit crowded in here...  Again, not germane to this...  

![Nothing to see here](https://media.giphy.com/media/13d2jHlSlxklVe/giphy.gif)